.circleci/config.yml

```diff
@@ -361,3 +390,32 @@ workflows:
               only: master
           requires:
               - npm_install
+  tap_tests:
+    triggers:
+      - schedule:
+          # Everyday @ 6PM UTC | 4PM EST | 1PM PST
+          cron: '0 18 * * 1-5'
+          filters:
+            branches:
+              only:
+                - master
+    jobs:
+      - npm_install:
+          filters:
+              branches:
+                only: master
+      - build:
+          requires:
+            - npm_install
+      - tap:
+          package: '@webex/internal-plugin-encryption'
+          requires:
+            - build
+      - tap:
+          package: '@webex/internal-plugin-conversation'
+          requires:
+            - build
+      - tap:
+          package: '@webex/internal-plugin-locus'
+          requires:
+            - build
```

Comment:
Are these the only ones defined in the scope of the implementation requirements?

Reply:
Nope, these are the ones the ones I remember (encryption & locus) or the ones we need (conversation)
I wanted you or Adam to provide more packages that directly touch services

Reply:
`webex-core` and `plugin-meetings` would be the two big ones we want to test as well.

Reply:
Update looks good! As discussed, our team should consider expanding these after a "trial" period of running just these packages.